viz-lib/src/visualizations/table/Renderer.jsx

```diff
@@ -103,7 +103,7 @@ export default function Renderer({ options, data, context }) {
     // so let's use this "hack" for better performance.
```

Comment:
@kravets-levko I wonder if this hack is needed at all... was it implemented following an actual issue or just a premature optimization?

Reply:
It was implemented to handle search in large tables. At that moment it looked like a good solution, however, now I guess it was possible to implement it in more clear way. If you want - I can revisit it, just please create an issue and assign it to me. Thanks!